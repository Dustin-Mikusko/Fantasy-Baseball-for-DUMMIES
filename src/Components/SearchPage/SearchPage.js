import React from 'react';
import { connect } from 'react-redux';
import './SearchPage.css';
import Header from '../Header/Header';
import SearchPlayerForm from '../SearchPlayerForm/SearchPlayerForm';

export const SearchPage = () => {
  return (
    <>
      <Header />
      <SearchPlayerForm />
    </>
  )
}

export default SearchPage;
