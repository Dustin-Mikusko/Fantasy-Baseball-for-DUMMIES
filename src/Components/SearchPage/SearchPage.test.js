import React from 'react';
import SearchPage from './SearchPage';
import { shallow } from 'enzyme';

describe('SearchPage', () => {
  it('should match the snapshot', () => {
    const wrapper =shallow(<SearchPage />);

    expect(wrapper).toMatchSnapshot();
  });
})
