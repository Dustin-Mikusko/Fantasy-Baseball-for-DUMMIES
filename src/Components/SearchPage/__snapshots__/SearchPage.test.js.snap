// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchPage should match the snapshot 1`] = `ShallowWrapper {}`;
